cmd/svcat/binding/bind_cmd.go

```diff
@@ -126,6 +143,12 @@ func (c *bindCmd) bind() error {
 		return err
 	}
 
+	if c.wait {
+		glog.V(2).Info("Waiting for binding to be injected...")
+		pollInterval := 1 * time.Second
+		binding, err = c.App.WaitForBinding(binding.Namespace, binding.Name, pollInterval, c.timeout)
+	}
+
 	output.WriteBindingDetails(c.Output, binding)
-	return nil
+	return err
```

Comment:
This deserves a comment that it is assumed err will be either nil from c.App.Bind or [nil or err] from c.App.WaitForBinding.

I would vote the more verbose way of using the pattern of if err ! nil, return err inside of c.wait and revert this change and make it clear that this return means bind() executed clean.